pytest_stoq/fixtures.py

```diff
@@ -51,6 +51,11 @@ def current_till(store, example_creator, current_station):
     return Till.get_current(store, current_station) or example_creator.create_till()
 
 
+@pytest.fixture
+def mock_new_store(monkeypatch, store):
```

Comment:
it would be nice to have tests for this. you can use the existing tests [1] as a base.

[1] https://github.com/stoq/pytest-stoq/blob/master/tests/test_fixtures.py